### ISSUE TITLE
Testing target values

### ISSUE THREAD

**Issue by Icedgarr** [jfloff/pywFM#15]
Hello

In order to run the model one needs to add a vector y of target values for the testing dataset also. I don't understand the requirement, why does it need the test values? Are they used when training the model?

____
Parameters
----------
x_train : {array-like, matrix}, shape = [n_train, n_features]
    Training data
y_train : numpy array of shape [n_train]
    Target values
x_test: {array-like, matrix}, shape = [n_test, n_features]
    Testing data
y_test : numpy array of shape [n_test]
    Testing target values
____
My idea is to predict the values of a test set from which I do not have the target values, how should I proceed?

**Comment by jfloff** [comment]
Hello,

`libfm` uses the test values to output some results regarding its predictions. They are not used when training the model. If I'm not mistaken, you could actually set them to a dummy value and just collect the predictions (just disregard the prediction statistics since those will be wrong).

Hope that's clear. For more info check [libfm manual](www.libfm.org/libfm-1.42.manual.pdf).
